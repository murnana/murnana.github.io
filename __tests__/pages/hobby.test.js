import React from "react"
import renderer from "react-test-renderer"
import HobbyPage from "../../src/pages/hobby"

describe("HobbyPage", () => {
  it("snapshot", () => {
    const tree = renderer.create(<HobbyPage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
