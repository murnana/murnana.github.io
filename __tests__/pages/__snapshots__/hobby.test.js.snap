// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HobbyPage snapshot 1`] = `
<div>
  <div>
    <h1>
      Hobby
    </h1>
    <p>
      趣味のページ
    </p>
  </div>
  <div>
    <ul>
      <li>
        <a
          href="/hobby/sdorica"
        >
          Sdorica
        </a>
      </li>
    </ul>
  </div>
</div>
`;
